examples.py

```diff
@@ -85,7 +91,7 @@ def evaluate_detection_example():
     classes = ['kite', 'person']
 
     # Image shape
-    mask_shape = (480, 640)
+    # mask_shape = (480, 640)
```

Comment:
Why did you comment some lines of code? If they are unused, I think we should delete them.